engines/order_management/spec/services/order_management/stock/coordinator_spec.rb

```diff
@@ -5,7 +5,12 @@
 module OrderManagement
   module Stock
     describe Coordinator do
-      let!(:order) { create(:order_with_line_items, distributor: create(:distributor_enterprise)) }
+      let!(:order) do
+        build_stubbed(
+          :order_with_line_items,
+          distributor: build_stubbed(:distributor_enterprise)
+        )
```

Comment:
can you briefly explain when we should use build_stubbed vs just build? I think that would deserve an entry on the wiki. If you post here I can add to the wiki, thanks!

Reply:
See below :wink:

Reply:
@luisramos0 The `build` strategy can create records for associations — it uses the `create` strategy for associations while the `build_stubbed` strategy uses `build_stubbed` for associations as well.

Reply:
awesome, thanks!